### DISCUSSION TITLE
Change limit StreamChannelListController

### DISCUSSION THREAD

**Discussion by EtcGonza** [GetStream/stream-chat-flutter#1136] [Q&A] [answered]
Greetings! I am using getStreamChat v4 and in my StreamChannelListView I set the controller a limit of 40 channels. So if I understand it right it should give me 40 channels when StreamChannelListView loads. But i noticed that the first call have a min function so it doesn´t matter what number put in the limit, the first time the limit always will be 30 and after the first load the controller will use the limit I set.

There is some way I can get my 40 channels on my first load using the StreamChannelListView?

**Comment by EtcGonza** [comment, marked as answer]
I create an issue and I thought I could remove this topic but can´t. Sorry!!

**Reply by xsahil03x** [reply]
The max limit at backend for the query Channels API is currently 30. So it's not possible to load more than 30 channels at the moment.

Thanks,
Sahil

**Reply by EtcGonza** [reply]
Thanks for the quick answer!
